### ISSUE TITLE
linux-disconnect.js doesn't disconnect due bad command

### ISSUE THREAD

**Issue by starfishmod** [elqui-app/node-wifi#9]
the current command generates
nmcli dev disconnect iface wlan0
if iface is set
this should be 
nmcli dev disconnect wlan0
change 
```
if (config.iface) {
    	    commandStr += " iface " + config.iface;
    	}
```

to
```
if (config.iface) {
    	    commandStr += " " + config.iface;
    	}
```

**Comment by thibaultfriedrich** [comment]
Ok i will be correct it  in the next version (later this day I think).

**Comment by thibaultfriedrich** [comment]
It has been fixed in version 1.2.3 now available on npm
